app/api.js

```diff
@@ -22,9 +24,22 @@ export function publishSiteState({ community, conference, events }) {
     },
     body,
   })
-  .then(response => {
-    if (response.status === 201) return response.json();
-    throw new Error('Unable to publush site content');
+  .then(assertStatusOK)
+  .then(parseJSON);
+}
+
+function get(url) {
+  return fetch(url, {
+    credentials: 'same-origin',
   })
-  .catch(error => { throw error; });
+  .then(assertStatusOK)
+  .then(parseJSON);
+}
+
+export function fetchSiteState() {
+  return get(SITE_ENDPOINT);
+}
+
+export function publishSiteState({ community, conference, events }) {
```

Comment:
This destructuring seems superfluous, as you're not doing anything with with the dissected variables, other than a straight passthrough into `post()`